Add project guidelines to Terms page

wafflecoffee opened the pull request:
This adds the project guidelines as outlined [here](https://discord.com/channels/734077874708938864/734077874708938867/806556531491471368) to the Terms page.

NOTE: I've made a few tweaks in wording to accommodate this format, so this is not an exact copy.
Among changes that I've made to Pros's original document (not fully comprehensive):
- "mod" reworded to "Project" if applicable
- Clarified that projects are content
- Some wording has been clarified (e.g. "auto pvp" to "automatic PvP" or "item duping" to "item duplication"
- Oxford comma added in at least one place

---
A deploy can be found at https://knossos-git-project-guidelines-emmaffle.vercel.app/legal/terms.

venashial commented:
Would putting the project guidelines on a separate page be a better way to go about this? Linking modders to a page with just the guidelines could make it more clear what exactly the guidlines are.

venashial commented:
Alternatively, you could add an `id` attribute to the `h2` to make anchor linking work. Like `https://knossos-git-project-guidelines-emmaffle.vercel.app/legal/terms#project_rules`

wafflecoffee commented:
> Would putting the project guidelines on a separate page be a better way to go about this? Linking modders to a page with just the guidelines could make it more clear what exactly the guidlines are.

I was going to do that, but it looked awful in the footer :P
![image](https://i.imgur.com/VIgOmPh.png)

> Alternatively, you could add an `id` attribute to the `h2` to make anchor linking work. Like `https://knossos-git-project-guidelines-emmaffle.vercel.app/legal/terms#project_rules`

I'll do that

venashial commented:
It doesn't have to be in the footer. It could just be linked to from the terms and from the mod creation page.

wafflecoffee commented:
> It doesn't have to be in the footer. It could just be linked to from the terms and from the mod creation page.

I've moved it to a separate page. I went against adding it to the mod creation page, as no matter how I implemented it, it seemed too "in your face"; I think a link from the terms page (which is what I've done) is sufficient.
https://knossos-git-project-guidelines-emmaffle.vercel.app/legal/terms
https://knossos-git-project-guidelines-emmaffle.vercel.app/legal/rules
Each of the subheadings has ids, too, so you can do, for example, [#misleading](https://knossos-git-project-guidelines-emmaffle.vercel.app/legal/rules#misleading).

Mysterious-Dev reviewed (approved):
LGTM
